dashboard/client/src/type/actor.ts

```diff
@@ -30,6 +30,7 @@ export type Actor = {
     [key: string]: number;
   };
   exitDetail: string;
+  reprName: string;
```

Comment:
Isn't this supposed to be optional? Or does it always exist?

Reply:
it will always exists, as an empty string if no __repr__ is defined on the actor.